src/VisualStudio/InteractiveServices/Interactive/VsInteractiveWindowPackage.cs

```diff
@@ -52,6 +52,8 @@ protected override void Initialize()
 
             _componentModel = (IComponentModel)GetService(typeof(SComponentModel));
             _interactiveWindowProvider = _componentModel.DefaultExportProvider.GetExportedValue<TVsInteractiveWindowProvider>();
+            KnownUIContexts.ShellInitializedContext.WhenActivated(() =>
+                _componentModel.GetService<HACK_ThemeColorFixer>());
```

Comment:
The "immediate" ```IClassificationFormatMap``` that we're updating is for the Immediate Window (eventually mapping down to [```DefGuidList.guidImmediateWindowFontCategory```](https://msdn.microsoft.com/en-us/library/vstudio/microsoft.visualstudio.editor.defguidlist.guidimmediatewindowfontcategory(v=vs.110).aspx).